_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
```

Comment:
Ideally there would be a method to "*" this where below you can mention (as you might remember from my talk) the durability is optional (default is off).